neurodsp/sim/aperiodic.py

```diff
@@ -238,6 +239,82 @@ def sim_powerlaw(n_seconds, fs, exponent=-2.0, f_range=None, **filter_kwargs):
 
     return sig
 
+@normalize
+def sim_fgn(n_seconds, fs, hurst=0.5):
+    """Simulate a fractional gaussian noise time series with specified Hurst exponent.
+
+    Parameters
+    ----------
+    n_seconds : float
+        Simulation time, in seconds.
+    fs : float
+        Sampling rate of simulated signal, in Hz.
+    hurst : float, optional, default: -0.5
```

Comment:
There seems to be some discrepancy on whether Hurst exponents are written as positive or negative. 

Here, the value should match the signature, so I think it should be 0.5 (without negative). 

Relatedly, the values in the `Examples sections` seem to use negative, when I think they should also be positive?

Reply:
Yes, thank you. The Hurst exponent should be positive. This is a typo.